config/protractor/protractor-dev.conf.js

```diff
@@ -113,7 +113,11 @@ if (process.env.TRAVIS) {
   config.capabilities = {
     browserName: 'chrome',
     'chromeOptions': {
-      'args': ['--disable-extensions --ignore-certificate-errors']
+      'args': [
+        '--disable-extensions',
+        '--ignore-certificate-errors',
+        '--no-sandbox'
```

Comment:
This `--no-sandbox` flag is the key to getting the build to run on Chrome 79 in Linux.

Reply:
Something so small... :-P